src/app/dashboard/export-overlay/export-overlay.component.html

```diff
@@ -71,7 +71,7 @@
     </div>
 
     <div class="capturing-parent">
-      <app-team></app-team>
+      <!-- <app-team></app-team> -->
```

Comment:
Do you need this later on?

Reply:
I'll fix this when I fix the image export